README.MD

```diff
@@ -20,28 +20,19 @@ It makes use of the API to request the following products and resources:
 
 ###Usabilla for Apps
   - Apps
-  - Feedback items 
+  - Feedback items
 
 For more information on resources, authorization and available API calls, please visit out [documentation](https://usabilla.com/api).
 
-Changes in version 1.2
-- Added In-Page resource
-
-Changes in version 1.1
-Resources are now retrieved by providing the following:
-  - A scope
-  - A product name
-  - A resource type
-
 # Installation:
-Requires Python 2.7. (Need Collections.OrderedDict)
+
+Requires Python 2.7
 
 ```bash
-python setup.py install
+pip install usabilla
 ```
 
-# Usage:
-## sample command:
+# Buttons example:
 ```python
 >>> import usabilla as ub
 >>> api = ub.APIClient('CLIENT-API-KEY', 'CLIENT-SECRET-KEY')
```

Comment:
The Usabilla APP, Public API interface refers to a access key and a secret key. Better to keep naming aligned to avoid confusion